### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by Thalmalynx** [sharsil/mailcat#25]
Hey guys,

I'm not a command prompt pro, far from it, but so far I've never had any problems (Holele, Toutatis, blackbird...). However, I have a little problem with MailCat.

It may come from the path: C:\Users\Me\mailcat\mailcat.py

I tried to run two scripts : 
- mailcat\mailcat.py USERNAME
- python mailcat/mailcat.py USERNAME

Same result: 

C:\Users\Me\mailcat\mailcat.py:1712: DeprecationWarning: There is no current event loop
  jobs = asyncio.gather(*[print_results(checker, target, req_session_fun, args.verbose) for checker in checkers])
C:\Users\ME\mailcat\mailcat.py:1714: DeprecationWarning: There is no current event loop
  asyncio.get_event_loop().run_until_complete(jobs)

Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Program Files\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] Une connexion existante a dû être fermée par l’hôte distant

Error while checking gmail: 421 Cannot connect to SMTP server 2a00:1450:4025:c03::1b (2a00:1450:4025:c03::1b:25), connect error 10060
Error while checking yandex: 421 Cannot connect to SMTP server 2a02:6b8::311 (2a02:6b8::311:25), connect error 10060
Error while checking mailDe: 421 Cannot connect to SMTP server 2001:868:100:600::211 (2001:868:100:600::211:25), connect error 10060

I guess it's a silly mistake on my part, can someone point me in the right direction?

**Comment by Thalmalynx** [comment]
Thank you for your quick response! 

1/ Python 3.10 (64-bit)

2/  Indeed, I get an error message with telnet : 
421 cannot connect to smtp server XXXXX error 10060

My IP address is not listed. 

![image](https://user-images.githubusercontent.com/106340785/196061263-6565fc40-cf79-4360-afe8-0df605e0f42e.png)

**Comment by Thalmalynx** [comment]
Yet it works very well with holehe, toutatis, maigret, blackbird, ignorant...

**Comment by imakealol** [comment]
Hi i have the same issue, with Python 3.11.0

thank you

**Comment by soxoj** [comment]
@Thalmalynx please, check mailcat version from this branch: https://github.com/sharsil/mailcat/tree/async-refactor

**Comment by C3n7ral051nt4g3ncy** [comment]
@Thalmalynx see PR #37

**Comment by soxoj** [comment]
TL;DR: If you get connection errors, then [decrease max connections count](https://github.com/sharsil/mailcat/blob/main/README.md#troubleshooting)
